docs/user/dashboard/aggregation-based.asciidoc

```diff
@@ -191,6 +191,27 @@ TIP: Aggregation-based panels support a maximum of three *Split series*.
 [role="screenshot"]
 image:images/aggBased_barChartTutorial2_8.4.png[Bar chart with sample logs data]
 
+[float]
+[[edit-agg-based-visualizations-in-lens]]
+==== Open and edit aggregation-based visualizations in Lens
+
+When you open aggregation-based visualizations in *Lens*, all configuration options appear in the *Lens* visualization editor.
+
+You can open the following aggregation-based visualizations in *Lens*:
+
+* Area
+* Gauge
+* Goal
+* Heat map
+* Hortizontal bar
+* Line
+* Metric
+* Vertical bar
+
+To get started, click *Edit visualization in Lens* in the toolbar.
```

Comment:
It is not on the toolbar, it is on the navigation panel (where inspect and save are)

<img width="538" alt="image" src="https://user-images.githubusercontent.com/17003240/205846663-2867b945-94c6-4f48-8165-8b6409e59c16.png">